app/scripts/templates/marketing_snippet_ios.mustache

```diff
@@ -23,16 +23,16 @@
           <p>
             {{#t}}Get the full Sync experience.{{/t}}
             <br>
-            {{#t}}Download Firefox for mobile devices.{{/t}}
+            <span>{{#t}}Download Firefox for mobile devices.{{/t}}</span>
           </p>
 
           <ul>
-            <li>
-                <a class="marketing-link marketing-link-android" href="https://app.adjust.com/2uo1qc?campaign=fxa-conf-page&adgroup=android&creative=button" data-marketing-id="spring-2015-android-ios-sync"><img width="152" height="45" src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
-            </li>
-            <li>
-                <a class="marketing-link marketing-link-ios" href="https://app.adjust.com/2uo1qc?campaign=fxa-conf-page&adgroup=ios&creative=button&fallback=https://itunes.apple.com/app/apple-store/id989804926?pt=373246&ct=adjust_tracker&mt=8" data-marketing-id="spring-2015-android-ios-sync"><img width="152" height="53" src="{{ appStoreImage }}" alt="{{#t}}Download Firefox for iOS!{{/t}}" /></a>
+          	<li>
```

Comment:
nit: can you ensure the spacing/indentation is as it was before? 2 spaces in for each line, it looks like this is only 1 space in, and the next is 3 in from here.